library/coretests/tests/floats/f128.rs

```diff
@@ -260,98 +249,98 @@ fn test_classify() {
 #[cfg(not(miri))]
```

Comment:
Miri has f16 and f128 implementations for all the rounding functions by now -- I don't think there's still a good reason for these tests to be disabled in Miri?

Reply:
Not at all, I just wasn't aware of the change. I added 2e4824d8b4a8b1a33908f7af702b07d8871de544 to remove the `#[cfg]`.

(which try job runs miri again?)

Reply:
> (which try job runs miri again?)

The gnu-aux one.